dissect/thumbcache/c_thumbcache.py

```diff
@@ -75,5 +75,4 @@
     uint32  Unknown;        // 0x28
 }; // 0x2C
 """
-c_thumbcache_index = cstruct()
-c_thumbcache_index.load(c_thumbcache_index_def)
+c_thumbcache_index = cstruct().load(c_thumbcache_index_def)
```

Comment:
```suggestion
c_thumbcache_index = cstruct().load(thumbcache_index_def)
```
Can you also rename the definition variable?